docs/development/tips_and_tricks.rst

```diff
@@ -131,7 +131,9 @@ Note that the ``cat`` example above will also add the ATHS info for the
 Architecture Diagrams
 ---------------------
```

Comment:
As a side note, this section is extremely buried in our docs right now, suggestions for reorganizing welcome for a follow-up PR :)